launch_testing/launch_testing/__init__.py

```diff
@@ -0,0 +1,62 @@
+import io
+import re
+from launch.output_handler import LineOutput
+
+
+class InMemoryHandler(LineOutput):
+
+    def __init__(
+        self, name, ld, expected_lines, regex_match=False,
+        filtered_prefixes=[
+            b'pid', b'rc',
+            b'RTI Data Distribution Service Evaluation License issued to',
+            b'Expires on']):
```

Comment:
I can't come up with a better idea either, so I've added a `TODO` that we can address in the future.